templates/repo/issue/labels/labels_selector_field.tmpl

```diff
@@ -21,8 +33,14 @@
 					<div class="divider"></div>
 				{{end}}
 				{{$previousExclusiveScope = $exclusiveScope}}
-				<a class="{{if .IsChecked}}checked{{end}} item" href="#" data-id="{{.ID}}" data-id-selector="#label_{{.ID}}" data-scope="{{$exclusiveScope}}"><span class="octicon-check {{if not .IsChecked}}gt-invisible{{end}}">{{if $exclusiveScope}}{{svg "octicon-dot-fill"}}{{else}}{{svg "octicon-check"}}{{end}}</span>&nbsp;&nbsp;{{RenderLabel $.Context .}}
-				{{if .Description}}<br><small class="desc">{{.Description | RenderEmoji $.Context}}</small>{{end}}</a>
+				<a class="{{if .IsChecked}}checked{{end}} item" href="#" data-id="{{.ID}}" {{if .IsArchived}}data-is-archived{{end}} data-id-selector="#label_{{.ID}}" data-scope="{{$exclusiveScope}}"><span class="octicon-check {{if not .IsChecked}}gt-invisible{{end}}">{{if $exclusiveScope}}{{svg "octicon-dot-fill"}}{{else}}{{svg "octicon-check"}}{{end}}</span>&nbsp;&nbsp;{{RenderLabel $.Context .}}
+					{{if .Description}}<br><small class="desc">{{.Description | RenderEmoji $.Context}}</small>{{end}}
+					{{if .IsArchived}}
+						<i class="archived-label-hint" data-tooltip-content={{ctx.Locale.Tr "archived"}}>
+							{{svg "octicon-info"}}
+						</i>
+					{{end}}
```

Comment:
Why not the normal archived label we should even have a template for?`

Reply:
that template contains a not complete 'archive' word and here I only want 'i' hover to show archived 

Thats why...

Reply:
But why?
Why introduce an inconsistency in the display?

Reply:
I have changed the code Now we are using the shared template all over this feature.